packages/web-components/src/components/content-block/content-block-paragraph.ts

```diff
@@ -20,9 +22,22 @@ const { stablePrefix: ddsPrefix } = ddsSettings;
  */
 @customElement(`${ddsPrefix}-content-block-paragraph`)
 class DDSContentBlockParagraph extends LitElement {
+  @internalProperty()
+  content = '';
+
+  firstUpdated() {
+    this.content = stripHTML(this.innerHTML);
+    render(
+      html`
+        <p>${this.content}</p>
```

Comment:
I'm wondering if adding the `<p>` tag is a good idea here, can we just run the `stripHTML` without adding additional wrapper tags? I think the `<p>` tag also wouldn't really bring any benefit from an SEO perspective.